src/java.base/share/native/libsyslookup/syslookup.c

```diff
@@ -25,9 +25,14 @@
 
 // Note: the include below is not strictly required, as dependencies will be pulled using linker flags.
 // Adding at least one #include removes unwanted warnings on some platforms.
-#include <stdlib.h>
 
+#include <stdlib.h>
+#include <math.h>
 // Simple dummy function so this library appears as a normal library to tooling.
 char* syslookup() {
-  return "syslookup";
+return "syslookup";
 }
+
+
+
+
```

Comment:
Please avoid changing lines which you don't need to modify! Is `#include <math.h>` really needed, here? If so, please protect it by `#ifdef _AIX` and add a comment explaining why.